### PR TITLE
Fixing SES5 links to documentation.suse.com (noref)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/admin_ceph_gateway.xml [hunk 1]
```diff
@@ -3757,7 +3757,8 @@ rgw_zone=us-west</screen>
   <para>
    You can use the &haproxy; load balancer to distribute all requests across
    multiple &ogw; back-end servers. Refer to
-   <link xlink:href="https://www.suse.com/documentation/sle-ha-12/book_sleha/data/sec_ha_lb_haproxy.html"/>
+   <link
+    xlink:href="https://documentation.suse.com/sle-ha/15-SP1/single-html/SLE-HA-guide/#sec-ha-lb-haproxy"/>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]: you refer to 15sp1 section here, is that intentional?

Comment (asettle) [reply xml/admin_ceph_gateway.xml]: Nope, accident. Will fix.

**Comment by lproven** [comment]
>  &ceph; block devices can integrate with the &qemu; virtual machines. For more
  information on &qemu; &kvm;, see

QEMU KVM? Should that not be "QEMU _and_ KVM" or something like that?

**Comment by lproven** [comment]
> https://documentation.suse.com/sles/12-SP5/single-html/SLES-admin/#cha-ntp

The anchor does not resolve for me -- it just goes to the top of the page.

**Comment by lproven** [comment]
>      <link xlink:href="https://documentation.suse.com/sles/12-SP5/single-html/SLES-admin/#sec-network-yast"/>

Seems to go to the top of the page.

Should it be...
https://documentation.suse.com/sles/12-SP5/single-html/SLES-admin/#sec-basicnet-yast
?

**Comment by lproven** [comment]
Apart from those 3 small points, LGTM.

**Comment by tbazant** [comment]
> > &ceph; block devices can integrate with the &qemu; virtual machines. For more
> > information on &qemu; &kvm;, see
> 
> QEMU KVM? Should that not be "QEMU _and_ KVM" or something like that?

No, QEMU KVM is correct, QEMU is a general emulator while KVM is sort of its plugin. without KVM, QEMU behaves very slowly as it has no acceleration support in the kernel.

**Comment by tbazant** [comment]
> except for my 1 comment, all links are valid.

Sorry that i did not check whether the anchors work, i'll do it next time :-)

**Pull request merged**